### ISSUE TITLE
Rune linting not supported in *.svelte.js files in Ubuntu

### ISSUE THREAD

**Issue by ahkelly** [sveltejs/eslint-plugin-svelte#890]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

9.13.0

### What version of `eslint-plugin-svelte` are you using?

2.46.0

### What did you do?

<details>
<summary>Configuration</summary>

WSL/Ubuntu 22
VS code latest (running in wsl)
Node 22

Default npx sv create install selecting JSDoc typing

@playwright/test@1.48.2
@sveltejs/adapter-auto@3.3.1
@sveltejs/kit@2.7.3
@sveltejs/vite-plugin-svelte@4.0.0
@types/eslint@9.6.1
eslint-config-prettier@9.1.0
eslint-plugin-svelte@2.46.0
eslint@9.13.0
globals@15.11.0
prettier-plugin-svelte@3.2.7
prettier@3.3.3
svelte-check@4.0.5
svelte@5.1.3
typescript@5.6.3
vite@5.4.10
vitest@2.1.3
```
</details>

```svelte
export let order_credits_amount = $state(0);
```

### What did you expect to happen?

The linter to support the new runes $state(), $derived(), etc

### What actually happened?

A red line and undefined is reported when a rune is used. This issue with exact same configuration on my mac does not have an issue - it appears to be a Linux issue (I have confirmed on 2 different workstations running Ubuntu in wsl).

![image](https://github.com/user-attachments/assets/f12031d8-96c7-4148-bafe-6f126df8d621)

### Link to **GitHub Repo** with Minimal Reproducible Example

Not applicable as this is an IDE related issue.

**Comment by ahkelly** [comment]
I gave the repro steps

1. npx sv create in WSL / Ubuntu 22
2. open vs code in WSL and create a shared.svelte.js file and create a $state() rune

This cannot be reproduced in the playground / repl as it is tied to a specific os

**Comment by ota-meshi** [comment]
I don't have Ubuntu. If we don't have a way to reproduce it, there's nothing we can do.
Please investigate the cause yourself.

**Comment by ahkelly** [comment]
May want to mark this plugin as

1. Not ready for svelte 5 (shortcuts are still svelte 4 only)
2. MacOS only supported
